src/test/resources/org/jenkinsci/plugins/oic/ConfigurationAsCode.yml

```diff
@@ -14,5 +14,6 @@ jenkins:
       groupsFieldName: groupsFieldName
       logoutFromOpenidProvider: true
       scopes: scopes
+      tokenAuthMethod: client_secret_post
```

Comment:
Add test case for wrong "tokenAuthMethod" value.

Reply:
I can find no examples that do this for CasC. If you show me how, I would be glad to add it.